vendor/github.com/v3io/v3io-go-http/container.go

```diff
@@ -2,6 +2,7 @@ package v3io
 
 import (
 	"sync/atomic"
+	"time"
```

Comment:
Is this vendor change an update from upstream or a custom edit?

Reply:
edited manually, but should be same as in vendor package

Reply:
Better check it out instead.

Reply:
(because we don't want manually cherry-picked vendor packages if we can avoid them)